en/docs/learn/extended-self-registration-api-and-account-confirmation-api.md

```diff
@@ -0,0 +1,473 @@
+        
+        **Sample Request**
+        ```
+        curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "+947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+        ```
+        
+        **Sample Response**
+        ```
+        {
+        "code": "USR-02001",
+        "message": "Successful user self registration. Pending account verification",
+        "notificationChannel": "SMS",
+        "confirmationCode": null
+        }
+        ```
+    
+    ??? info "User Self Registers with Verified Claims"
+        **Sub scenario:** The user self registers with verified claims. 
+        
+        If the verified claims match the preferred notification channel, 
+        the user will be created and the account will be unlocked (HTTP: 201). 
+        
+        **Sample Request**
+        ```
+        curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user": {"username": "kim","realm": "PRIMARY", "password": "Password12!","claims": [{"uri": "http://wso2.org/claims/givenname","value": "kim" },{"uri": "http://wso2.org/claims/emailaddress","value": "kim.anderson@gmail.com"},{"uri": "http://wso2.org/claims/identity/emailVerified","value": "true"},{"uri": "http://wso2.org/claims/lastname","value": "Anderson"},{"uri": "http://wso2.org/claims/mobile","value": "+947721584558"} ] },"properties": []}' "https://localhost:9443/api/identity/user/v1.0/me"
+        ```
+        
+        **Sample Response**
+        ```
+        {
+        "code": "USR-02004",
+        "message": "Successful user self registration with verified channel. Account verification not required.",
+        "notificationChannel": null,
+        "confirmationCode": null
+        }
+        ```
+        
+        !!! note
+            - If the verified claims do not match the preferred notification channel, the user will be 
+            created and the user will be asked to verify the account via the preferred channel. 
+            
+            - **Example**: User's `SMS` channel is already verified but user wants `EMAIL` to be the preferred 
+            channel of communication.
+            
+            **Sample Response**
+            ```
+            {
+            "code": "USR-02001",
+            "message": "Successful user self registration. Pending account verification",
+            "notificationChannel": "EMAIL",
+            "confirmationCode": null
+            }
+            ```
+
+2. **Self registration with notifications are externally managed and the account is locked on creation.** 
```

Comment:
Shall we change this to "Self registration with externally managed notifications and account lock on creation"?

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549